### PR TITLE
test(ingest/minio): Configure delta lake minio tests for arm64

### PR DESCRIPTION
**Pull request by asikowitz**
Replaces with docker container
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-ingestion/tests/integration/delta_lake/minio/setup_minio.sh [hunk 1]
```diff
@@ -1,6 +1,15 @@
+INTEGRATIONS_DIR="./tests/integrations/minio"
 #ref https://hub.docker.com/r/minio/minio/
-
-wget https://dl.min.io/server/minio/release/linux-amd64/minio -P ./tests/integrations/delta_lake/minio/
-chmod +x ./tests/integrations/delta_lake/minio/minio
-nohup ./tests/integrations/delta_lake/minio/minio server ./tests/integrations/delta_lake/minio/data > temp.log 2>&1 &
-echo $! > ./tests/integrations/delta_lake/minio/minio_pid.txt
+if [[ $(uname -m) == 'arm64' && $(uname) == 'Darwin' ]]; then
```
Comment (treff7es) [inline metadata-ingestion/tests/integration/delta_lake/minio/setup_minio.sh:3]: Why we don't install with brew if the arch is `Darwin` and `amd64`?

FYI -> here is how we do brew install in the preflight script -> 
https://github.com/datahub-project/datahub/blob/cfa864e4187b6f9adafd7e951294be8e6109d03d/metadata-ingestion/scripts/datahub_preflight.sh#L8C7-L8C7

File: metadata-ingestion/tests/integration/delta_lake/minio/setup_minio.sh [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
+  if ! command -v minio &> /dev/null; then
+    brew install minio/stable/minio
+  fi
+  mkdir -p "${INTEGRATIONS_DIR}"
+  MINIO_CMD="minio"
+else
+  wget https://dl.min.io/server/minio/release/linux-amd64/minio -P "${INTEGRATIONS_DIR}"
```
Comment (treff7es) [inline metadata-ingestion/tests/integration/delta_lake/minio/setup_minio.sh:10]: What about this?
```suggestion
  wget https://dl.min.io/server/minio/release/$(uname)-$(uname -m)/minio -P "${INTEGRATIONS_DIR
```

File: metadata-ingestion/tests/integration/delta_lake/minio/setup_minio.sh (hunk 1 above)
Comment (asikowitz) [reply metadata-ingestion/tests/integration/delta_lake/minio/setup_minio.sh:3]: I was just trying to make work with arm macs. I don't really have a preference here

**Comment by hsheth2** [comment]
@asikowitz to update this to use docker instead of downloading binaries

**Review by hsheth2 — APPROVED** [review]
This looks way better

**Pull request merged**